x/ibc_query/keeper/keeper.go

```diff
@@ -102,117 +102,123 @@ func (k Keeper) SetNextQuerySequence(ctx sdk.Context, sequence uint64) {
 }
 
 // SetSubmitCrossChainQuery stores the MsgSubmitCrossChainQuery in state keyed by the query id
```

Comment:
Could you change function name in comment?? 
The function name in comment is old version..